iocore/net/SSLNetVConnection.cc

```diff
@@ -991,8 +994,6 @@ SSLNetVConnection::free(EThread *t)
   }
   con.close();
 
-  ats_free(tunnel_host);
```

Comment:
This part was moved into `TLSTunnelSupport::_clear()`. It should be ok as we call the function in `clear()` on line 1009.

Reply:
agreed.